zerver/lib/markdown/help_relative_links.py

```diff
@@ -57,6 +48,41 @@ def gear_handle_match(key: str) -> str:
     return gear_instructions.format(item=item)
 
 
+help_info = {
+    # The pattern is key: [name, link]
+    # key is from REGEXP: `{relative|help|key}`
+    # name is what the item is called in the help menu: `Select **name**.`
+    # link is used for relative links: `Select [name](link).`
+    "keyboard-shortcuts": [
+        '<i class="zulip-icon zulip-icon-keyboard"></i> Keyboard shortcuts',
+        "/#keyboard-shortcuts",
+    ],
+    "message-formatting": [
+        '<i class="zulip-icon zulip-icon-edit"></i> Message formatting',
+        "/#message-formatting",
+    ],
+    "search-filters": [
+        '<i class="zulip-icon zulip-icon-manage-search"></i> Search filters',
+        "/#search-operators",
+    ],
+}
+
+help_instructions = """
+1. Click on the **Help menu** (<i class="zulip-icon zulip-icon-help"></i>) icon
+   in the upper right corner of the web or desktop app.
```

Comment:
What if we put the instructions in a Desktop/Web tab, and simplify the first step? This would be enough:

> Click on the **Help menu** (<i class="zulip-icon zulip-icon-help"></i>) icon
   in the upper right corner of the app.